traffic_ops/testing/api/v2/withobjs_test.go

```diff
@@ -99,6 +100,7 @@ var withFuncs = map[TCObj]TCObjFuncs{
 	ServerChecks:                         {CreateTestServerChecks, DeleteTestServerChecks},
 	ServerServerCapabilities:             {CreateTestServerServerCapabilities, DeleteTestServerServerCapabilities},
 	Servers:                              {CreateTestServers, DeleteTestServers},
+	ServiceCategories:					  {CreateTestServiceCategory, DeleteTestServiceCategories},
```

Comment:
gofmt

Reply:
is there tests cases related to these? looks like it was added to the json fixtures but nothing else?